serverless-plugin/index.js

```diff
@@ -42,6 +33,23 @@ class ServerlessPlugin {
    * Modify the CloudFormation template before the package is finalized
    */
   finalizeHook () {
+    const { topicArn, ...pluginConfig } = (
+      this.serverless.service.custom || {}
+    ).slicWatch
+    if (!topicArn) {
+      throw new ServerlessError('topicArn not specified in custom.slicWatch')
+    }
```

Comment:
Do we still need this or is it something we can make mandatory in the JSON schema validation?